src/shared/components/virtualized-log-viewer/useTokenizationWorker.ts

```
@@ -0,0 +1,193 @@
+  private handleWorkerMessage(event: MessageEvent<TokenizeResponse>) {
+    const { id } = event.data;
+    const pending = this.pendingRequests.get(id);
+
+    if (pending) {
+      this.pendingRequests.delete(id);
+      pending.resolve(event.data);
+    }
+  }
+
+  private handleWorkerError(error: ErrorEvent) {
+    console.error('Worker error:', error);
```

ESLint

[error] Unexpected console statement.

(no-console)